packages/apsara-ui/src/Modal/Modal.tsx

```diff
@@ -24,7 +24,7 @@ const Modal = ({
     maskStyle = { background: "rgba(0 0 0 / 0.5);" },
 }: ModalProps) => {
     return (
-        <Dialog.Root open={open} onOpenChange={closable ? onClose : () => {}}>
+        <Dialog.Root open={open} onOpenChange={closable ? onClose : () => ({})}>
```

Comment:
The previous value in the else condition is `() => void` and the modified one is `() => object`, is this intended? If we don't want to pass anything if `closable` is false, can we use `closable ? onClose : undefined` instead?

Reply:
replaced `() => object` with `() => void`